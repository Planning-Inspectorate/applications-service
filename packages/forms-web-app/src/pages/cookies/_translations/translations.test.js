const enCookiesTranslations = require('./en.json');
const cyCookiesTranslations = require('./cy.json');

describe('pages/cookies/_translations', () => {
	it('should return the correct English translations', () => {
		expect(enCookiesTranslations).toMatchSnapshot();
	});

	it('should return the correct Welsh translations', () => {
		expect(cyCookiesTranslations).toMatchSnapshot();
	});

	it('should return the same keys for English and Welsh translations', () => {
		expect(enCookiesTranslations).toHaveSameKeys(cyCookiesTranslations);
	});
});
