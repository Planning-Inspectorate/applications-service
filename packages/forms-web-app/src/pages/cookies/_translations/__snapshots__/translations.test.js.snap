// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`pages/cookies/_translations should return the correct English translations 1`] = `
{
  "errorMessage": {
    "listItem1": "reloading the page",
    "listItem2": "turning on JavaScript in your browser",
    "paragraph1": "We use JavaScript to set most of our cookies. Unfortunately Javascript is not running on your browser, so you cannot change your settings. You can try:",
  },
  "heading1": "Cookies on the Find a National Infrastructure Project service",
  "heading2": "Cookie settings",
  "heading3": "Strictly necessary cookies",
  "heading4": "Cookies used to improve the Find a National Infrastructure Project service",
  "legend1": "Do you want to accept analytics cookies?",
  "listItem1": "remember your progress through our service",
  "listItem2": "remember you have seen the cookies message",
  "paragraph1": "Cookies are files saved on your phone, tablet or computer when you visit a website. We use cookies to store information about how you use the Find a National Infrastructure Project service, such as the pages you visit.",
  "paragraph2": "We use different types of cookies.",
  "paragraph3": "You can choose which cookies you're happy for us to use and how the data is shared.",
  "paragraph4": "We use essential cookies to help you use the Find a National Infrastructure Project service. These do things such as:",
  "paragraph5": "These cookies need to be turned on below to give you the best experience using the service.",
  "paragraph6": "We use Google Analytics to measure how you use the Find a National Infrastructure Project service so we can improve it based on user needs. We do not allow Google to use or share the data about how you use this site.",
  "paragraph7": "You agree to the Planning Inspectorate using your data to help improve the Find a National Infrastructure Project service.",
  "successBanner": {
    "heading1": "Your cookie settings were saved",
    "linkText1": "Go back to the page you were looking at",
    "paragraph1": "Government services may set additional cookies and, if so, will have their own cookie policy and banner.",
  },
}
`;

exports[`pages/cookies/_translations should return the correct Welsh translations 1`] = `
{
  "errorMessage": {
    "listItem1": "ail-lwytho'r dudalen",
    "listItem2": "troi ymlaen Javascript yn eich porwr",
    "paragraph1": "Rydym yn defnyddio JavaScript i osod y rhan fwyaf o'n cwcis. Yn anffodus, nid yw JavaScript yn rhedeg ar eich porwr, felly ni allwch newid eich gosodiadau. Gallwch roi cynnig ar:",
  },
  "heading1": "Cwcis ar y gwasanaeth Dod o Hyd i Brosiect Seilwaith Cenedlaethol",
  "heading2": "Gosodiadau cwcis",
  "heading3": "Cwcis cwbl angenrheidiol",
  "heading4": "Cwcis a ddefnyddir i wella'r gwasanaeth Dod o Hyd i Brosiect Seilwaith Cenedlaethol",
  "legend1": "Ydych chi eisiau derbyn cwcis dadansoddi?",
  "listItem1": "cofio'ch cynnydd trwy ein gwasanaeth",
  "listItem2": "cofio eich bod wedi gweld y neges ynghylch cwcis",
  "paragraph1": "Ffeiliau sy'n cael eu harbed ar eich ffôn, eich llechen neu eich cyfrifiadur pan fyddwch yn ymweld â gwefan yw cwcis. Rydym yn defnyddio cwcis i storio gwybodaeth am y ffordd rydych yn defnyddio'r gwasanaeth Dod o Hyd i Brosiect Seilwaith Cenedlaethol, fel y tudalennau yr ymwelwch â nhw.",
  "paragraph2": "Rydym yn defnyddio gwahanol fathau o gwcis.",
  "paragraph3": "Gallwch ddewis pa gwcis rydych yn fodlon i ni eu defnyddio a sut mae'r data'n cael ei rannu.",
  "paragraph4": "Rydym yn defnyddio cwcis hanfodol i'ch helpu i ddefnyddio'r gwasanaeth Dod o Hyd i Brosiect Seilwaith Cenedlaethol. Mae'r rhain yn gwneud pethau fel:",
  "paragraph5": "Mae angen i'r cwcis hyn gael eu derbyn isod er mwyn rhoi'r profiad gorau i chi wrth ddefnyddio'r gwasanaeth.",
  "paragraph6": "Rydym yn defnyddio Google Analytics i fesur sut rydych yn defnyddio'r gwasanaeth Dod o Hyd i Brosiect Seilwaith Cenedlaethol fel y gallwn ei wella yn seiliedig ar anghenion defnyddwyr. Nid ydym yn caniatáu i Google ddefnyddio na rhannu'r data am y ffordd rydych yn defnyddio'r safle hwn.",
  "paragraph7": "Rydych yn cytuno i'r Arolygiaeth Gynllunio ddefnyddio eich data i helpu i wella'r gwasanaeth Dod o Hyd i Brosiect Seilwaith Cenedlaethol.",
  "successBanner": {
    "heading1": "Arbedwyd eich gosodiadau cwcis",
    "linkText1": "Dychwelyd i'r dudalen yr oeddech yn edrych arni",
    "paragraph1": "Fe allai gwasanaethau'r llywodraeth osod cwcis ychwanegol ac, os felly, bydd ganddynt eu polisi a'u banner cwcis eu hunain.",
  },
}
`;
